test/box-luatest/gh_4726_box_execute_before_cfg_test.lua

```diff
@@ -0,0 +1,15 @@
+local t = require('luatest')
+local g = t.group()
+
+g.test_box_execute_before_box_cfg = function()
+    t.assert_equals(rawget(box, 'execute') == nil, true)
```

Comment:
Nit: I would rather use `t.assert_equals(rawget(box, 'execute'), nil)`. This way we'll get an actual value on the screen if the test case will fail.